listenbrainz_spark/config.py.sample

```diff
@@ -50,3 +50,7 @@ LOG_SENTRY = {
 # String + UUID
 # The following var defines the string of which the model id is made up of.
 MODEL_ID_PREFIX = 'listenbrainz-recommendation-model'
+
+FTP_SERVER_URI = 'ftp.musicbrainz.org'
```

Comment:
I think we should use ftp.eu.metabrainz.org since that mirror is in the EU, where most of our servers are.